homeassistant/components/iqvia/__init__.py

```diff
@@ -151,22 +152,22 @@ def unit_of_measurement(self):
         """Return the unit the value is expressed in."""
         return "index"
 
+    @callback
+    def _handle_coordinator_update(self) -> None:
+        """Handle updated data from the coordinator."""
+        self.update_from_latest_data()
+        self.async_write_ha_state()
+
     async def async_added_to_hass(self):
```

Comment:
just one thought if we are only calling super this whole method can be removed right?